package.json

```diff
@@ -90,25 +90,19 @@
       "/.module-cache/",
       "/react/build/"
     ],
-    "persistModuleRegistryBetweenSpecs": true,
     "rootDir": "",
     "scriptPreprocessor": "scripts/jest/preprocessor.js",
-    "setupEnvScriptFile": "scripts/jest/environment.js",
+    "setupFiles": [ "scripts/jest/environment.js" ],
```

Comment:
Can you lose the spaces? They'll get wiped away when something like `npm install -S` happens anyway so let's avoid the future noise.